docs/Data/Semiring/Boolean.md

```diff
@@ -0,0 +1,28 @@
+## Module Data.Semiring.Boolean
+
+#### `BooleanSemiring`
+
+``` purescript
+newtype BooleanSemiring a
+  = BooleanSemiring a
+```
+
+Provides the semiring for any boolean algebra.
```

Comment:
Or, of course, I'd be happy to just note the which one this is in the documentation. Someone else could separately add a newtype that inverts a boolean algebra as well I suppose, and then use it with this...